resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,151 @@
+        required: true
+        style: simple
+        schema:
+          pattern: ^[0-9]{4}-[0-9]{2}$
+          type: string
+          example: 2016-17
+      requestBody:
+        description: ''
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/NationalInsuranceTestSupportSaAnnualSummaryrequest'
+              - example:
+                  scenario: HAPPY_PATH_1
+            example:
+              scenario: HAPPY_PATH_1
+        required: true
+      responses:
+        '201':
+          description: ''
+          headers: {}
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/NationalInsuranceTestSupportSaAnnualSummaryresponse'
+                - example:
+                    class1:
+                      totalNICableEarnings: 10
+                    class2:
+                      totalDue: 20
+                    maxNICsReached: false
+              examples:
+                example-1:
+                  description: Example response for scenario HAPPY_PATH_1
+                  value:
+                    class1:
+                      totalNICableEarnings: 10
+                    class2:
+                      totalDue: 20
+                    maxNICsReached: false
+                example-2:
+                  description: Example response for scenario HAPPY_PATH_2
+                  value:
+                    class1:
+                      totalNICableEarnings: 17
+                    class2:
+                      totalDue: 319.57
+                    maxNICsReached: true
+      deprecated: false
+components:
+  schemas:
+    errorResponse:
+      title: errorResponse
+      required:
+      - code
+      type: object
+      properties:
+        code:
+          type: string
+    Class1:
+      title: Class1
+      required:
+      - totalNICableEarnings
+      type: object
+      properties:
+        totalNICableEarnings:
+          type: number
+          description: Total Class 1 earnings subject to National Insurance contributions between the Primary Threshold and the Upper Earnings Limit
+          example: 10
+    Class2:
+      title: Class2
+      required:
+      - totalDue
+      type: object
+      properties:
+        totalDue:
+          type: number
+          description: Total Class 2 National Insurance contributions due
+          example: 10
+    NationalInsuranceTestSupportSaAnnualSummaryrequest:
+      title: NationalInsuranceTestSupportSaAnnualSummaryrequest
+      type: object
+      properties:
+        scenario:
+          oneOf:
+          - $ref: '#/components/schemas/Scenario'
+          - $ref: '#/components/schemas/Scenario1'
+          description: The scenario to create test data for
+      description: Create National Insurance summary test data request
+    NationalInsuranceTestSupportSaAnnualSummaryresponse:

```

Comment:
Please change to camel case i.e. `nationalInsuranceTestSupportSaAnnualSummaryResponse`. Thanks